lib/src/base-styles/_nx-colors.scss

```diff
@@ -16,9 +16,9 @@
   --nx-color-form-element-border-pristine: var(--nx-swatch-indigo-90);
 
   --nx-color-interactive-background-selected: var(--nx-swatch-blue-90);
-  --nx-color-interactive-background-active: var(--nx-swatch-blue-95);
-  --nx-color-interactive-background-hover: var(--nx-swatch-indigo-95);
-  --nx-color-interactive-background-disabled: var(--nx-swatch-grey-95);
+  --nx-color-interactive-background-active: var(--nx-swatch-grey-95);
+  --nx-color-interactive-background-hover: var(--nx-swatch-grey-90);
+  --nx-color-interactive-background-disabled: var(--nx-swatch-grey-90);
```

Comment:
I don't think you'll require these changes once you get latest changes from the feature branch.

Reply:
> I don't think you'll require these changes once you get latest changes from the feature branch.

Yep, I merged the feature branch before I saw this comment, and looking at the diffs its cleared these changes.